src/gdm/temporal_models.py

```diff
@@ -0,0 +1,39 @@
+from datetime import date
+
+from gdm import DistributionComponentBase
+from pydantic import field_validator
+from infrasys import Component
+from uuid import UUID
+
+
+class DeletionModel(Component):
+    name: str = ""
+    component_uuid: UUID
+
+
+class AdditionModel(Component):
+    name: str = ""
+    component: DistributionComponentBase
```

Comment:
Why does this have the actual component whereas the others have the component UUID?